packages/frontend/pages/index.module.scss

```diff
@@ -1 +1,14 @@
 @import "../styles/variables";
+
+.particles {
+	z-index: 0;
+	position: fixed;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+}
+
+.canvas {
+
+}
```

Comment:
What is this for?

Reply:
I was going to put a z-index property in case the foreground elements gets behind the background in the future. I removed the canvas class name for now.

Reply:
Nice, okay!